pkg/operation/care/checker.go

```diff
@@ -166,7 +166,10 @@ func (b *HealthChecker) checkEtcds(condition gardencorev1beta1.Condition, object
 			)
 
 			if lastError := object.Status.LastError; lastError != nil {
-				codes = gardencorev1beta1helper.ExtractErrorCodes(gardencorev1beta1helper.DeprecatedDetermineError(errors.New(*lastError)))
+				configurationProblemRegexp := regexp.MustCompile(`(?i)(KubeletHasInsufficientMemory|KubeletHasDiskPressure|KubeletHasInsufficientPID)`)
```

Comment:
Can you explain when the `lastError` of an `Etcd` would contain such messages?

Reply:
It was added with https://github.com/gardener/gardener/pull/2242, My assumption was that the PR introduced possible errors and used `DetermineErrorCode` where it could have`.

Reply:
I think those messages can only apply to nodes (https://github.com/gardener/gardener/pull/2242/files#diff-06ea2c044593e68973be932ce9d497c714d2e5b0fb4b89f147d24e3d1fb77542R171-R176).